integration/airflow/openlineage/airflow/macros.py

```diff
@@ -41,12 +41,15 @@ def lineage_parent_id(run_id: str, task: "BaseOperator", task_instance: "TaskIns
     PythonOperator(
         task_id='render_template',
         python_callable=my_task_function,
-        op_args=['{{ lineage_parent_id(run_id, task, task_instance) }}'], # macro invoked
+        op_args=['{{ lineage_parent_id(task_instance) }}'], # macro invoked
         provide_context=False,
         dag=dag
     )
     """
-    job_name = OpenLineageAdapter.build_task_instance_run_id(
-        task_instance.dag_id, task.task_id, task_instance.execution_date, task_instance.try_number
+    return "/".join(
+        [
+            _DAG_NAMESPACE,
+            f"{task_instance.dag_id}.{task_instance.task.task_id}",
```

Comment:
Just to be sure, i think we should use `integration/airflow/openlineage/airflow/utils.get_job_name` - the same method as used in the listener.